src/Metable.php

```diff
@@ -180,6 +194,20 @@ public function removeMeta(string $key): void
         $this->getMetaCollection()->pull($key)->delete();
     }
 
+    /**
+     * Delete many `Meta` keys.
+     *
+     * @param array|string[]|Traversable $keys
+     *
+     * @return void
+     */
+    public function removeManyMeta($keys): void
+    {
+        foreach ($keys as $key) {
+            $this->removeMeta($key);
+        }
```

Comment:
This is an N+1. If adding a method that implies bulk operations, it should probably be efficient under the hood. Could probably do a delete with a `whereIn()` and clean up the relationship collection manually.

Reply:
That's a very good point. I hadn't considered large sets and the effect it would have on the process. One thing I noticed is that `removeMeta()` throws an exception if the meta doesn't exist on the model. Is that intentional?

I ended up doing this in my project, which definitely feels really heavy:

```php
public function removeManyMeta(array $keys): void
{
    foreach ($keys as $key) {
        if ($this->hasMeta($key)) {
            $this->removeMeta($key);
        }
    }
}
```

The `whereIn()` would resolve that issue in the remove-many method, but the issue would still be there in `removeMeta()`.

I'll refactor this to be more efficient 👍🏻

Reply:
ah, that error case is not intentional. We could fix that by checking for the hasMeta internally.